.browserslistrc

```diff
@@ -0,0 +1,6 @@
+chrome >= 55
+edge >= 15
+firefox >= 53
+opera >= 42
+safari >= 11
+ios >= 11
```

Comment:
please add a new line here. every file should have one - otherwise, we'll get unwanted changes if the IDE adds one

```suggestion
ios >= 11

```